recipe/meta.yaml

```diff
@@ -14,20 +10,19 @@ source:
   md5: d04fba2d9245e661f245de0577f48a33
 
 build:
-  number: {{ build_number }}
-  string: {{ build_string }}
+  number: 8
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
-    - libpng {{ libpng_version }}
+    - libpng >=1.6.28,<1.7
     - zlib 1.2.*
     - xz 5.2.*
     - toolchain
 
   run:
-    - libpng {{ libpng_version }}
+    - libpng >=1.6.23,<1.7
```

Comment:
This will not work. Please change to `libpng >=1.6.28,<1.7` according to https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L48

More info for you to read: 
https://github.com/conda-forge/matplotlib-feedstock/issues/41
https://github.com/conda-forge/matplotlib-feedstock/pull/64
https://github.com/conda-forge/conda-forge.github.io/pull/245e